dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScopeTest.kt

```diff
@@ -6246,6 +6246,108 @@ internal class RumViewScopeTest {
 
     // endregion
 
+    // region Feature Flags
+
+    @Test
+    fun `M send event W handleEvent(AddFeatureFlagEvaluation) on active view`(
+        @StringForgery flagName: String,
+        @StringForgery flagValue: String
+    ) {
+        // GIVEN
```

Comment:
Empty `GIVEN`. Should it be like that?

Reply:
All the givens here are set up by the fixture setup. I'm not sure if I should keep the comment and have a blank body or remove the comment?